Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "arangors-graph-exporter"
-version = "0.0.8"
+version = "0.0.9"
```

i assume this pr warrants a 0.0.10 release

```suggestion
version = "0.0.10"
```

---

Yes, actually I was quite surprised that main is not on 0.0.9. We should implement a release pipeline/process asap.